resolwe_bio/processes/workflows/wgbs.yml

```diff
@@ -0,0 +1,50 @@
+- slug: workflow-wgbs
+  category: workflow:primary
+  name: Whole genome bisulfite sequencing
+  data_name: "Pipeline ({{ reads|sample_name|default('?') }})"
+  requirements:
+    expression-engine: jinja
+  version: 0.0.1
+  type: data:workflow:wgbs
+  input:
+    - name: reads
+      label: Input reads
+      type: data:reads:fastq
+    - name: genome
+      label: genome
+      type: data:genome:fasta
+    - name: genome_identifier
+      label: The UCSC Genome Browser identifier
+      type: basic:string
+      allow_custom_choice: true
```

Comment:
I am not clearly sure how this parameter is used in the tool code, but the code is working normally if you put other genomes_indentifier for example rn6.